docs/tutorial.rst

```diff
@@ -147,7 +147,7 @@ Step 5. Create and attach a runner to the repository.
 
 .. code-block:: bash
 
-    cd /misc/fernlab1/misc/
+    cd /path/to/your/dockerimage/directory
```

Comment:
Not for the docker image, that one is not locally saved in disk, but the repository clone. Maybe just remove dockerimage from the path.
```
/path/to/your/directory/
```